app/talk/popular-tags.cjsx

```diff
@@ -0,0 +1,35 @@
+React = require 'react'
+PromiseRenderer = require '../components/promise-renderer'
+{makeHTTPRequest} = require 'json-api-client'
+{Link} = require 'react-router'
+
+module?.exports = React.createClass
+  displayName: 'TalkPopularTags'
+
+  propTypes:
+    header: React.PropTypes.object             # Optional header ex: <h1>Tags</h1>
+    section: React.PropTypes.string.isRequired # Talk section
+    params: React.PropTypes.object.isRequired # URL params
+
+  tagsRequest: ->
+    popularTags = talkClient.root + "/tags/popular?section=#{@props.section}&limit=10"
+    makeHTTPRequest 'GET', popularTags, null, talkClient.headers
```

Comment:
Looks like this is a fairly unaddressed issue with the spec:

https://github.com/json-api/json-api/issues/648

Since I really don't care about dogmatically following a spec, what in particular is tripping up your client?